### ISSUE TITLE
pgadmin-6.13 failed to build

### ISSUE THREAD

**Issue by boozedog** [NixOS/nixpkgs#194094]
### Steps To Reproduce
1. build pgadmin.service

### Build log
```
building the system configuration...
error: builder for '/nix/store/aavsrczk8s801zcj7pbg6zb4nkkf2za3-python3.10-azure-mgmt-subscription-3.1.1.drv' failed with exit code 1;
       last 10 log lines:
       > adding 'azure_mgmt_subscription-3.1.1.dist-info/RECORD'
       > removing build/bdist.linux-x86_64/wheel
       > Finished executing setuptoolsBuildPhase
       > installing
       > Executing pipInstallPhase
       > /build/azure-mgmt-subscription-3.1.1/dist /build/azure-mgmt-subscription-3.1.1
       > Processing ./azure_mgmt_subscription-3.1.1-py3-none-any.whl
       > ERROR: Could not find a version that satisfies the requirement msrest>=0.7.1 (from azure-mgmt-subscription) (from versions: none)
       > ERROR: No matching distribution found for msrest>=0.7.1
       > 
       For full logs, run 'nix log /nix/store/aavsrczk8s801zcj7pbg6zb4nkkf2za3-python3.10-azure-mgmt-subscription-3.1.1.drv'.
error: 1 dependencies of derivation '/nix/store/1xh6nhaignb2b3ki63573kpiclrz9681-pgadmin-6.13.drv' failed to build
error: 1 dependencies of derivation '/nix/store/bjgf7hdhj0s17scajrfpvcb42k7009zp-unit-pgadmin.service.drv' failed to build
error: 1 dependencies of derivation '/nix/store/6fbf3ra8b5xciw86hcglxqf5xiiddfsp-system-units.drv' failed to build
error: 1 dependencies of derivation '/nix/store/w5qhr7z3lnjbprkk3i83aj9jlww8ph1m-etc.drv' failed to build
error: 1 dependencies of derivation '/nix/store/75zb40yvxb1bk5giv4lby333afcfh2jh-nixos-system-dab-xps-nixos-22.11.20220930.10ecda2.drv' failed to build
```

### Notify maintainers
@gador 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
Built from flake using nixos-unstable
```

**Comment by boozedog** [comment]
OK this is interesting, I get same error when I try to build from `github:gador/nixpkgs/2cf300385873504337a1af0502d9d10f8f4613df`

**Comment by gador** [comment]
fixed in current master. This is due to an update in `azure-mgmt-subscription`. The related PR is #193926. Also `pgadmin` got an update to 6.14 and a fix due to another python update see #192615 

Please see [here](https://nixpk.gs/pr-tracker.html?pr=193926) and [here](https://nixpk.gs/pr-tracker.html?pr=192615) as to when those get merged to `unstable`

**Comment by boozedog** [comment]
OK thanks @gador I'll keep using `github:gador/nixpkgs/e5061559f3a7cc1382e980c5c35126725cb965a7` til those get merged to unstable.
